with-lib/client-secret-basic-example/app.js

```diff
@@ -0,0 +1,40 @@
+var createError = require('http-errors');
```

Comment:
This file is pretty much the same on every example. Is there any way to unify and reuse it?

Reply:
i will remove it is not very useful